Source/WebCore/html/canvas/CanvasStyle.h

```diff
@@ -30,31 +30,28 @@
 #include "CanvasPattern.h"
 #include "Color.h"
 #include "ColorSerialization.h"
+#include <optional>
 #include <variant>
+#include <wtf/text/WTFString.h>
 
 namespace WebCore {
 
 class Document;
 class GraphicsContext;
 class CanvasBase;
```

Comment:
Not alphabetically sorted.

Reply:
Oh, that wtf/text/WTFString is unneeded! I was confused by an error and through that in and forgot. Will fix both.